qemu/tests/cfg/block_commit.cfg

```diff
@@ -68,3 +68,21 @@
             expected_speed = 1024000
             when_start = "set_speed stop resume pause_job resume_job cancel start"
             after_finished = "verify_backingfile reboot verify_alive"
+        - with_installation:
+            type = block_commit_installation
+            need_install = yes
+            start_vm = no
+            images = "stg"
+            source_image = "stg"
+            image_name_stg = "images/live_commit_installation"
+            image_size_stg = 20G
+            boot_drive_stg = yes
+            medium = cdrom
+            installation = cdrom
+            kernel = vmlinuz
+            initrd = initrd.img
+            force_create_image_stg = yes
+            image_aio = threads
+            unattended_delivery_method = cdrom
+            cdroms += " unattended"
```

Comment:
Add unattended cdrom only is not enough, if you want to use unattended iso , "cdrom_unattended" need be defined for different guest, such as `images/win10-64/autounattend.iso`
and "unattended_delivery_method = cdrom" will replaced by "floppy" that defined in Windows.cfg in staf.
Currently, it use answer.vfd, not unattended.iso, so L149~150 can be removed.

If you don't want use floppy, more cfg need be added to make unattended cdrom work.

Reply:
@yanan-fu Thanks, as we discussed, let's keep it with floppy installation for now, and move to cdrom when the cdrom format issue fixed.